.github/workflows/po-refresh.yml

```diff
@@ -7,6 +7,43 @@ on:
   # can be run manually on https://github.com/cockpit-project/cockpit/actions
   workflow_dispatch:
 jobs:
+  pot-upload:
```

Comment:
Fine for me as an incremental step to separate the privileges. Eventually I'd prefer this as a standalone pot-refresh.yml workflow, so that it can be run independently, scheduled daily, and more clearly structures what's going on.

Reply:
I actually think it makes sense for both of them to run as separate jobs inside of the same workflow, triggered at the same time, but I'm happy to discuss that further.  The manual trigger argument might be a good one, for example.

Reply:
For translators it makes sense to get more frequent POT updates, otherwise they'd have up to a week delay between us landing a new string and them being able to translate it. OTOH we don't want daily PO refresh PRs, that's just too much noise. For that reason, and because they do independent things, I think two separate workflows would be cleaner.